reports/Washington.md

```diff
@@ -935,3 +935,20 @@ id: wa-seattle-63
 * https://twitter.com/gramsofgnats/status/1301945336391368704
 * https://www.thestranger.com/slog/2020/09/04/44418202/nine-marchers-arrested-after-shutting-down-i-5
 
+
+### Police use bikes to ride into and shove protesters | September 7th
+
+Protesters gathered outside the Seattle Police Officers Guild. Police rode their bikes into protesters, then used their bikes to shove protesters back. No provocation on the part of protesters can be seen.
+
+Police allegedly used pepper spray and stun grenades while making 22 arrests.
+
+tags: protester, bike, arrest, stun-grenade, pepper-spray, shove, push
```

Comment:
Add "spray" here. Just an FYI when there's impact munitions or pepper spray, I try to use the verb and noun ("spray" & "pepper spray")